### PR TITLE
Update o-ran-integration.md

### PR DESCRIPTION
**Pull request by NMSVishal**
Added NFO REST API details which are used for OAI NF deployments.

### CODE REVIEW

File: content/en/docs/network-architecture/o-ran-integration.md [hunk 1]
```diff
@@ -123,7 +123,26 @@ NF Deployment Lifecycle Management services implemented using Nephio capabilitie
 {{% alert title="Note" color="primary" %}}
 
 In R3 Nephio use cases and component architectures have been defined for the Deploy O-RAN NF use case.
+In R3 ,NFO supports OAI O-RAN NF deployments using below REST API's.
+##  input will be packagevariant of OAI NF.
```
Comment (CsatariGergely) [inline content/en/docs/network-architecture/o-ran-integration.md:127]: I do not think that this should be a H2 heading. If you would like to highlight this just use `*` or `**` around the sentence.

File: content/en/docs/network-architecture/o-ran-integration.md [hunk 2]
```diff
@@ -123,7 +123,26 @@ NF Deployment Lifecycle Management services implemented using Nephio capabilitie
...
 
+##### NFO API For OAI DU NF Deployment:
```
Comment (CsatariGergely) [inline content/en/docs/network-architecture/o-ran-integration.md:129]: I would not use a heading here. Maybe a bulleted list is a better option.

File: content/en/docs/network-architecture/o-ran-integration.md [hunk 3]
```diff
@@ -123,7 +123,26 @@ NF Deployment Lifecycle Management services implemented using Nephio capabilitie
...
+
+    POST: BASE_URL/nfo/api/v1/operator/du/
+    GET: BASE_URL/nfo/api/v1/operator/du/
+    DELETE: BASE_URL/nfo/api/v1/operator/du/
+
+##### NFO API For OAI CU_CP NF Deployment:
```
Comment (CsatariGergely) [inline content/en/docs/network-architecture/o-ran-integration.md:135]: I would not use a heading here. Maybe a bulleted list is a better option.

File: content/en/docs/network-architecture/o-ran-integration.md [hunk 4]
```diff
@@ -123,7 +123,26 @@ NF Deployment Lifecycle Management services implemented using Nephio capabilitie
...
+
+    POST: BASE_URL/nfo/api/v1/operator/cucp/
+    GET: BASE_URL/nfo/api/v1/operator/cucp/
+    DELETE: BASE_URL/nfo/api/v1/operator/cucp/
+
+##### NFO API For OAI CU_UP NF Deployment:
```
Comment (CsatariGergely) [inline content/en/docs/network-architecture/o-ran-integration.md:141]: I would not use a heading here. Maybe a bulleted list is a better option.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

File: content/en/docs/network-architecture/o-ran-integration.md (hunk 4 above)
Comment (NMSVishal) [reply content/en/docs/network-architecture/o-ran-integration.md:141]: Sure ,I will change it to normal text

**Review by NMSVishal — COMMENTED** [review]
Removed h2 header format from NFO API Details section as suggested by @CsatariGergely

**Comment by NMSVishal** [comment]
@CsatariGergely  ,I have modified the

**Pull request closed without merging**